examples/auditLogging/server.js

```diff
@@ -0,0 +1,48 @@
+const express = require('express')
+const { makeExecutableSchema } = require('graphql-tools')
+
+const { ApolloVoyagerServer, gql } = require('../../packages/apollo-voyager-server')
+const { setAuditLogEnabled, wrapResolversForAuditLogging} = require('../../packages/apollo-voyager-audit')
+
+// This is our Schema Definition Language (SDL)
+const typeDefs = gql`
+  type Query {
+    hello: String
+  }
+`
+
+// Resolver functions. This is our business logic
+let resolvers = {
+  Query: {
+    hello: (obj, args, context, info) => {
+      return `Hello world from ${context.serverName}`
+    }
+  }
+}
+
+setAuditLogEnabled(true)
```

Comment:
Line 23/24: I guess we are working now to get that into core and pass config?

Reply:
actually I don't...
as we spoke during the chat today, I went with the simplest and quickest way to progress on cards.

Reply:
That is no problem. We could do that in no time after this is merged.